index.js

```diff
@@ -48,6 +76,9 @@ var TitleBar = function(options) {
 util.inherits(TitleBar, events.EventEmitter);
 
 TitleBar.prototype.appendTo = function(target) {
+	if (this._options.os == 'mac') var style = styleMac;
+	else if (this._options.os == 'win') var style = styleWin;
```

Comment:
I was having some trouble figuring out where to put that check. It's a little convoluted with the way the style and dom are appended.